src/fe/fe_boundary.C

```diff
@@ -639,7 +639,7 @@ void FEMap::compute_face_map(int dim, const std::vector<Real> & qw,
 
         // We need to look back at the full edge to figure out the normal
         // vector
-        const Elem * elem = side->parent();
+        const Elem * elem = side->interior_parent();
```

Comment:
So... "side->parent" with "side->interior_parent" throughout? I think we're pretty consistent about naming sides "side" but just curious if you also searched for all possible `foo->parent()` calls as well?

Reply:
Not a text replace, rather a manual search, but oh, no, I didn't search for *every* ```->parent()``` call, I'm afraid.  But we'll leave parent() set in default non --disable-deprecated builds for a couple years, and hopefully people will play with --disable-deprecated frequently enough in development to catch any incompatibilities before deployment.

I wish I could figure out how to make those incompatibilities into nice clean libmesh_deprecated() errors, but at least a nullptr dereference most often leads to a segfault plus stack trace rather than a corrupt solution.